app/src/interfaces/list-m2m/index.ts

```diff
@@ -14,14 +14,16 @@ export default defineInterface({
 	localTypes: ['m2m'],
 	group: 'relational',
 	options: ({ relations }: ExtensionsOptionsContext) => {
+		const relatedCollection = relations.o2m?.collection ?? relations.m2o?.related_collection;
```

Comment:
I have copied the code from `displays/relative-values/index.ts` that seems to work properly.

Reply:
```suggestion
		const relatedCollection = relations.m2o?.related_collection ?? relations.o2m?.collection;
```

To be compatible with https://github.com/directus/directus/pull/9391/files#diff-dcb0b7b8e24638261fb003d9ae4222a524a0da95c57c9a5997f0a65134dfe202R20

Reply:
The related collection for a m2m field should always be the o2m relation collection. We can remove the m2o one here

Reply:
Fixed